### DISCUSSION TITLE
How to set the correct shading for 0.3.11 in blender?

### DISCUSSION THREAD

**Discussion by jaredshao** [thygate/stable-diffusion-webui-depthmap-script#179] [Q&A]
I followed the readme method which worked in the old version, but 0.3.11's obj shows a lot of noise

![20230413230040](https://user-images.githubusercontent.com/7553200/231804387-7bcc006d-8c57-4f29-a660-68933152c342.png)

**Comment by thygate** [comment]
Might be something wrong with the obj perhaps, does the video generation work with this mesh ?

Nothing has changed in the inpainted mesh generation btw ..

**Reply by thygate** [reply]
You can also try this way https://github.com/thygate/stable-diffusion-webui-depthmap-script/issues/175#issuecomment-1506124967

**Reply by jaredshao** [reply]
Video generation works fine, problem in blender 3.5, haven't tried other versions

**Reply by thygate** [reply]
See also https://github.com/thygate/stable-diffusion-webui-depthmap-script/discussions/181#discussioncomment-5610697

**Comment by thygate** [comment]
https://github.com/thygate/stable-diffusion-webui-depthmap-script/discussions/181#discussioncomment-5618028
